### DISCUSSION TITLE
Support languages other than English

### DISCUSSION THREAD

**Discussion by dubadub** [cooklang/spec#144] [General]

**Comment by FlyinPancake** [comment]
Some languages, like Hungarian stick stuff together. For example "with milk" would become "tejjel" (tej meaning milk). Now this situation is managable by the spec (since I can just `@tej{}jel`), but sometimes this conjugation changes the root of the word also changes. For example apple is "alma" and whit apple is "almával" so i can't easily cut in the middle of the word, since "almá" makes no sense by itself (a and á are different sounds, not like accent in Spanish)
